client/lib/application/src/application-lifecycle/components/Loading.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ThreeColumns from 'common/src/components/pure/ThreeColumns.jsx';
+import Icon from 'react-fa';
+
+const Loading = () => {
+    return (
+        <ThreeColumns
```

Comment:
Ok, but I think `Loading` is not that special, so we can possibly split it up? Ie. one for reuse in `common` and one with columns in `application-lifecycle`?